### ISSUE TITLE
Support for multiple connections

### ISSUE THREAD

**Issue by laynetrain** [neo4j-contrib/django-neomodel#17]
I'm thinking about how support for multiple connections/ graph databases could work. I plan to have 1 graph per organization in a VPC. However, a user that logs in can belong to many organizations.

One way, I suppose, would be to have each neo4j db running on a different port, and then passing multiple values into `NEOMODEL_NEO4J_BOLT_URL =` Hmm. How then I could specify why graph a particular model method or query is dealing with? Would I preface each request with `django.conf.settings(NEOMODEL_NEO4J_BOLT_URL: my_port`?

Am I overthinking this? Do I just need to add a node for each organization, and make it truly multi-tenant?

**Comment by laynetrain** [comment]
TLDR; one graph per subdomain

**Comment by robinedwards** [comment]
Hey there is currently no way of supporting this in neomodel at present. But as you are going to run multiple neo4j instances on different ports is there any reason why you couldnt run multiple django instances with different configuration?

**Comment by laynetrain** [comment]
Hmm, so each django instance would have a different `config.DATABASE_URL = 'bolt://neo4j:password@localhost:7687'` where the port # varies?
